saleor/plugins/webhook/tests/subscription_webhooks/payloads.py

```diff
@@ -8,6 +8,25 @@
 from .....product.models import Product
 
 
+def generate_account_events_payload(customer_user, channel):
+    return json.dumps(
+        {
+            **generate_customer_payload(customer_user),
+            **{
+                "token": "token",
+                "redirectUrl": "http://www.example.com?token=token",
+                "channel": {
+                    "slug": channel.slug,
+                    "id": graphene.Node.to_global_id("Channel", channel.id),
+                },
+                "shop": {
+                    "domain": {"host": "mirumee.com", "url": "http://mirumee.com/"}
```

Comment:
```suggestion
                    "domain": {"host": "example.com", "url": "http://example.com/"}
```

Reply:
this needs to overwrite settings, so I would keep this as it is

Reply:
Fine, we will have to update it everywhere at some point, then.